README.md

```diff
@@ -50,8 +50,10 @@ or
 # make bootstrap-pkg-full
 ```
 
-(in the latter case please that `make bootstrap-pkg-full` only unpacks packages
-but does not build those of them that require compilation).
+(in the latter case please note that `make bootstrap-pkg-full` only unpacks packages
+but does not build those of them that require compilation. You can change to the
+`pkg` directory and then call `../bin/BuildPackages.sh` from there to build as many
+packages as possible).
```

Comment:
Looks good to me, although I would remove the parenthesis around this block.

Reply:
OK - I've removed them.